container.go

```diff
@@ -754,3 +775,37 @@ func (c *Container) detachDevices() error {
 
 	return nil
 }
+
+func (c *Container) addResources() error {
+	//TODO add support for memory, Issue: https://github.com/containers/virtcontainers/issues/578
+	if c.config == nil {
+		return nil
+	}
+
+	vCPUs := ConstraintsToVCPUs(c.config.Resources.CPUQuota, c.config.Resources.CPUPeriod)
+	if vCPUs != 0 {
```

Comment:
We should add an additional check here, looking for the current amount of CPUs on the VM. If the amount available is equal or higher than what is needed by the container (the variable `vCPUs` in this case), then we have nothing to do here.